README.md

```diff
@@ -41,7 +41,7 @@ pip install mteb
 * Using a python script (see [scripts/run_mteb_english.py](https://github.com/embeddings-benchmark/mteb/blob/main/scripts/run_mteb_english.py) and [mteb/mtebscripts](https://github.com/embeddings-benchmark/mtebscripts) for more):
 
 ```python
-from mteb import MTEB
+import mteb
```

Comment:
The CLI (in `cmd.py`) uses the same process to run tasks, I think it needs to be updated too

Reply:
I have created another PR for this as there were a few additional points that we might want to discuss further.